src/App.jsx

```diff
@@ -0,0 +1,44 @@
+import { Routes, Route } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import Navbar from './components/Navbar';
+import HousesList from './components/HousesList';
+import Reservation from './components/Reservation';
+import MyReservations from './components/MyReservations';
+import HouseReservations from './components/HouseReservations';
+import AddHouse from './components/AddHouse';
+import DeleteHouse from './components/DeleteHouse';
+import ShowHouse from './components/ShowHouse';
+import LoginForm from './components/LoginForm';
+import Splash from './components/Splash';
+import Logout from './components/Logout';
+import SignupForm from './components/SignupForm';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry.  

![image](https://github.com/jmonto55/book-an-appointment-frontend/assets/57421823/2403c1e3-f6fa-4479-baac-9d1b2fa26453)


That way, your code will meet the latest standards. 💪

Reply:
Noted 👍🏾